src/hotkey/command_executor.cpp

```diff
@@ -361,6 +362,7 @@ bool command_executor::do_execute_command(const hotkey::ui_command& cmd, bool pr
 			quit_confirmation::quit_to_desktop();
 			break;
 		case HOTKEY_QUIT_GAME:
+			gui2::switch_theme(prefs::get().gui2_theme());
```

Comment:
I guess this is also there to change the theme back when some scenario changed it?

Reply:
Yes.